packages/angular/projects/clr-angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -61,13 +61,8 @@ export const commonStringsDefault: ClrCommonStrings = {
   datepickerCurrentDecade: 'Current decade',
```

Comment:
Do we also need to remove the datagrid aria strings (line #48 - #51)?

Reply:
If you are talking about this lines 

```ts
  singleSelectionAriaLabel: 'Single selection header',
  singleActionableAriaLabel: 'Single actionable header',
  detailExpandableAriaLabel: 'Toggle more row content',
  datagridFilterAriaLabel: 'Toggle column filter',
```
I think that not, there used inside the `datagrid.html` as aria-labels for some of the header buttons also has there own `@Input` for overwritting them if needed.